app/views/shared/_landing_strip.html.haml

```diff
@@ -2,4 +2,5 @@
   = t('landing_strip_html',
     name: content_tag(:span, display_name(account), class: :emojify),
     domain: site_hostname,
+    root_path: root_path,
```

Comment:
Sorry if I may misunderstand your intention.
I think your intetion is:

`link_to_root_path: link_to(content_tag(:strong, site_hostname), root_path),`

`landing_strip_html: <strong>%{name}</strong> is a user on %{link_to_root_path}. You can follow them or interact with them if you have an account anywhere in the fediverse. If you don't, you can <a href="%{sign_up_path}">sign up here</a>.`

Right?